yt_dlp/extractor/patreon.py

```diff
@@ -406,10 +428,9 @@ def _get_comments(self, post_id):
             cursor = None
             for comment in traverse_obj(response, (('data', ('included', lambda _, v: v['type'] == 'comment')), ...)):
```

Comment:
The real problem is that the traversal path here is incorrect, thus the filtering it's trying to do is ineffective. Let's change it to this:
```suggestion
            for comment in traverse_obj(response, (('data', 'included'), lambda _, v: v['type'] == 'comment' and v['id'])):
```